pydantic/_internal/_validators.py

```diff
@@ -334,43 +333,25 @@ def max_length_validator(x: Any, max_length: Any) -> Any:
 def _extract_decimal_digits_info(decimal: Decimal) -> tuple[int, int]:
     """Compute the total number of digits and decimal places for a given [`Decimal`][decimal.Decimal] instance.
 
-    This function handles both normalized and non-normalized Decimal instances.
-    Example: Decimal('1.230') -> 4 digits, 3 decimal places
-
     Args:
         decimal (Decimal): The decimal number to analyze.
 
     Returns:
         tuple[int, int]: A tuple containing the number of decimal places and total digits.
```

Comment:
```suggestion
        tuple[int, int]: A tuple containing the number of decimal places and total digits.

        Though this could be divided into two separate functions, the logic is easier to follow if we couple the computation
    of the number of decimals and digits together.
```